iac/terraform/aws/auth/lambdas.tf

```diff
@@ -1,43 +1,49 @@
 locals {
+  lambda_archive_path = "${path.module}/../../../../core/services/aws/.build/zips"
   lambda_options = {
     refresh-token = {
       name     = "refresh-token"
-      zip_path = "${path.module}/../../../../core/services/aws/.build/zips/refreshToken.zip"
-      policy   = data.aws_iam_policy_document.lambda_common_policy
+      zip_path = "${local.lambda_archive_path}/refreshToken.zip"
       handler  = "refreshToken.default"
+      statement = [
```

Comment:
This is not good. So multiple lambdas which will use same policy can not share policy.

Reply:
every lambda will have its own statement block

Reply:
Isn't sharing policy better with reusable terraform code?

Reply:
yes, we can create local variables with policies. and we can use them in multiple places. this way we don't have to write duplicate codes

Reply:
like this, its this looks ok
![image](https://github.com/user-attachments/assets/61e34b94-669f-48ad-86f7-225ae31d2699)